uint32 a
uint32 b
---
GeometricSimilarity result
